client/components/Item/RelatedItems.vue

```diff
@@ -25,7 +25,7 @@
             :to="getItemDetailsLink(relatedItem)"
           >
             <v-list-item-avatar>
-              <v-img ref="avatarImg" :src="getImageUrl(relatedItem.Id)" />
+              <blurhash-image :item="relatedItem" :type="'Primary'" />
```

Comment:
```suggestion
              <blurhash-image :item="relatedItem" />
```
This is not needed because it's the default.